pingsweep.py

```diff
@@ -1,5 +1,55 @@
+# Sweep a range of IP addresses
+
+help_text = """ 
+pingsweep.py <https://github.com/Fallible/SimpleScripts>
+Flags
+ -h or --help
+	description:
+		prints this help text
+
+ --address <address>
+	description:
+ 		first 3 sections of ipv4 address
+ 	defaults to:
+ 		--address 10.11.1
+	
+--from <number>
+	description:
+		indicates start of sweep range
+	defaults to:
+		--from 1
+
+--to <number>
+	description:
+		indicates end of sweep range (non inclusive)
+	defaults to:
+		--to 255
```

Comment:
Looks like this needs to say 256.

Reply:
Instead of going to 256 I made it inclusive to match the expected behavior